module/documents/actor/actor.mjs

```diff
@@ -2694,6 +2694,7 @@ export default class Actor5e extends Actor {
     dhp = Number(dhp);
     const tokens = this.isToken ? [this.token?.object] : this.getActiveTokens(true);
     for ( const t of tokens ) {
+      if ( !t.isVisible ) continue;
```

Comment:
I would use the same logic from core. `isVisible` is too much overhead here I think. 
```suggestion
      if ( !t.visible || !t.renderable ) continue;
```